.github/workflows/check-prdoc.yml

```diff
@@ -47,10 +47,25 @@ jobs:
           echo "Labels: ${labels}"
           echo "labels=${labels}" >> "$GITHUB_OUTPUT"
 
-      - name: Validate prdoc for PR#${{ github.event.pull_request.number }}
+      - name: Get PR number
         if: ${{ github.event.pull_request.number != '' && !contains(steps.get-labels.outputs.labels, 'R0') }}
         run: |
-          echo "Validating PR#${GITHUB_PR}"
+          original_pr_number=''
+          PR_TITLE="${{ github.event.pull_request.title }}"
+          if [[ $PR_TITLE =~ 'Backport' ]]; then
```

Comment:
One comment here please like:  
Extract the numbers from titles in the form: ...